Epsilon.Host.Frontend/src/components/TopNavigation.vue

```diff
@@ -110,4 +103,12 @@ watch([fromDate, toDate], () => {
 		object-fit: contain;
 	}
 }
+
+#dateRangeCol {
+	margin-right: 30px;
+}
+
+.search-boxes {
+	margin-right: 105px;
```

Comment:
This moves the element to much. It's bad practice. It will not be responsive. And small changes will break the design quickly